src/Locator/GitRepositoryDirLocator.php

```diff
@@ -36,7 +39,22 @@ public function locate(string $gitDir): string
         $gitRepositoryDir = $matches[1];
 
         if ($this->filesystem->isAbsolutePath($gitRepositoryDir)) {
-            return $gitRepositoryDir;
+            if (!$this->filesystem->isFile($gitRepositoryDir.DIRECTORY_SEPARATOR.'commondir')) {
+                throw new RuntimeException('The git directory for worktree could not be found.');
```

Comment:
Is this expected behaviour? Or should we just link to the absolute dir instead?
Either way : we'll need a test for this

Reply:
For now, if the current repository is not recognized as worktree, just fallback to default locating method. Previously I was thinking about throwing an exception, as this would mean that something is very wrong with the repository (the file `commondir` must be available inside `.git/worktrees/worktree1`), but we may as well ignore it. I think it is a fine assumption.